common/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <!-- Camera permission is required for QR + PIN authorization method -->
+    <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
Is QR + PIN feature exclusive to broker, or is this also applicable to oneauth + MSAL?

I'm asking because this would be displayed on all consuming app's permission that they now require access to camera. (for those who are not aware of this QR+Pin feature... it could sound scary)

Reply:
this is exclusive to broker so I would need to move this permission to broker my b.

Reply:
I mean it would still show up on the Play Store page 
i.e.

![Uploading Screenshot 2024-01-25 at 12.42.22 PM.png…]()

Reply:
both teams are informed about this change.